scripts/validation_of_LIMS_upgrade.py

```diff
@@ -47,9 +47,9 @@
 from pprint import pprint
 from genologics.lims import *
 from genologics.config import BASEURI, USERNAME, PASSWORD
-import objectsDB as DB
+import LIMS2DB.objectsDB.objectsDB as DB
 from datetime import date
-lims = Lims(BASEURI, USERNAME, PASSWORD)
+lims = Lims('https://genologics.scilifelab.se:8443', USERNAME, PASSWORD)
```

Comment:
Maybe not so good to hardwire URLs? No big issue, since it won't change, but maybe move to config at some point.